yaraneba/spec/requests/task_request_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'rails_helper'
+
+RSpec.describe 'Tasks', type: :request do
+  before do
+    @task = FactoryBot.create(:task)
+  end
+
+  describe '#task index' do
+    context 'GET' do
+      example 'request OK' do
+        get root_path
+        expect(response.status).to eq(200)
+      end
+    end
+  end
+
+  describe '#task new' do
+    context 'GET' do
+      example 'request OK' do
+        get new_task_path
+        expect(response.status).to eq(200)
+      end
+    end
+    context 'POST' do
+      example 'request OK' do
+        post tasks_path, params: { task: FactoryBot.attributes_for(:task) }
+        expect(response.status).to eq(302)
+      end
+      example 'create task' do
+        expect do
+          post tasks_path, params: { task: FactoryBot.attributes_for(:task) }
+        end.to change{ Task.count }.by(1)
+      end
+    end
+  end
+
+  describe '#task edit' do
+
+    context 'GET' do
+      example 'request OK' do
+        get "/#{@task.id}/edit"
```

Comment:
routes の機能を使った方が良さそうです。

```suggestion
        get edit_task_path(task)
```